### PR TITLE
Koincity: roadmap Q3 (2024) and logo update

### PR DESCRIPTION
**Pull request by MiXiBo**
Resolves 

## Brief description
Koincity: 
+ Roadmap Q3 (2024)
+ Logo update

## Checklist

- [x ] I have built this pull request locally
- [x ] I have ran the unit tests locally
- [x ] I have manually tested this pull request
- [x ] I have reviewed my pull request
- [x ] I have added any relevant tests

### CODE REVIEW

**Comment by MiXiBo** [comment]
@sgerbino Let me know if I shall adjust something or if it's OK like that.

File: components/sections/Roadmap.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ export default function Roadmap() {
             { text: "Lords Forsaken Go-Live", icon: "fa-solid fa-gamepad" },
             { text: "Veive Protocol", icon: "fa-solid fa-wallet" },
             { text: "Koinos Marketing Campaign", icon: "fa-solid fa-business-time" },
+            { text: "Koincity: Staking Interface, Tax Token Creator, Low Value Token Converter", icon: "fa-solid fa-building" },
```
Comment (sgerbino) [inline components/sections/Roadmap.js:97]: Can we come up with something a little more succinct?

**Review by sgerbino — CHANGES_REQUESTED** [review]

**Comment by sgerbino** [comment]
I like the new logo and landing page, nicely done.

**Comment by MiXiBo** [comment]
Thanks! I will try to shorten the description. I don't want to add more lines

**Review by sgerbino — APPROVED** [review]

**Pull request merged**